README.md

```diff
@@ -172,8 +172,10 @@ onto a Docker container and used to create a virtual environment capable of runn
 The plugin will automatically inject logic for building and using this virtual environment into a pre-existing Dockerfile
 by default. The `dockerfile` configuration must be set to the target Dockerfile. To disable the Dockerfile update
 altogether, set the `updateDockerfile` configuration to `false`. Because the final image must match the image that builds
-the virtual environment, Habushu must know what the base image for the final output should be. By default, `python:3.11-slim`
-is used, but this can be customized with the `dockerBase` and `dockerUser` configurations.
+the virtual environment, Habushu defaults to using `python:3.11` to build the virtual environment and  `python:3.11-slim`
```

Comment:
`Because the final image must match the image that builds the virtual environment`. This is no longer the case right?

Reply:
Reworded this section more significantly to put the problem of compatible venv up front.  Made this part more clear as a result.